recipe/meta.yaml

```diff
@@ -49,7 +50,7 @@ about:
   home: https://github.com/kennethreitz/responder/
   license: APACHE-2.0
   license_family: APACHE
-  license_file: LICENSE
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
IIUC this is unnecessary as more recent conda builds check this directory by default

Reply:
sure, let's try a suggested change 😉

Reply:
Touché!

Reply:
looks like AppVeyor is not happy with suggested changes, had to rebuild the PR manually...

Reply:
Yeah, saw that - wasn't sure what was going on.

Other than that "suggested changes" seem pretty awesome!

I'm a big fan of the [papercuts](https://blog.github.com/2018-08-28-announcing-paper-cuts/) project.

Reply:
looks cool!